src/Cli/func/StaticResources/node-v4-templates.json

```
@@ -614,5 +614,47 @@
       "enabledInTryMode": false,
       "userPrompt": []
     }
+  },
+  {
+    "id": "McpToolTrigger-Typescript-4.x",
+    "runtime": "2",
+    "files": {
+      "%functionName%.ts": "import { app, InvocationContext, output } from \"@azure/functions\";\n\nconst PROPERTY_TYPE = \"string\";\n\n// Hello function - responds with hello message\nexport async function mcpToolHello(_toolArguments:unknown, context: InvocationContext): Promise<string> {\n    // Get name from the tool arguments\n    const mcptoolargs = context.triggerMetadata.mcptoolargs as {\n        name?: string;\n    };\n    const name = mcptoolargs?.name;\n\n    console.info(`Hello ${name}, I am MCP Tool!`);\n\n    return `Hello ${name}, I am MCP Tool!`;\n}\n\n// Register the hello tool\napp.mcpTool('hello', {\n    toolName: 'hello',\n    description: 'Simple hello world MCP Tool that responses with a hello message.',\n    toolProperties:[\n    {\n      propertyName: \"name\",\n      propertyType: PROPERTY_TYPE,\n      description: \"Required property to identify the caller.\",\n      isRequired: true,\n    }],\n    handler: mcpToolHello\n});"
+    },
+    "metadata": {
+      "defaultFunctionName": "mcpToolTrigger",
+      "description": "$McpToolTrigger_description",
+      "name": "Mcp Tool Trigger",
```

Should we call this MCP as it is an acronym? 

```suggestion
      "name": "MCP Tool Trigger",
```

---

Done